Networking/Networking/Remote/WCPayRemote.swift

```diff
@@ -35,6 +35,30 @@ public class WCPayRemote: Remote {
 
         enqueue(request, mapper: mapper, completion: completion)
     }
+
+    /// Captures a payment for an order. See https://stripe.com/docs/terminal/payments#capture-payment
+    /// - Parameters:
+    ///   - siteID: Site for which we'll capture the payment.
+    ///   - orderID: Order for which we are capturing the payment.
+    ///   - paymentIntentID: Stripe Payment Intent ID created using the Terminal SDK.
+    ///   - completion: Closure to be run on completion.
+    public func captureOrderPayment(for siteID: Int64,
+                               orderID: Int64,
+                               paymentIntentID: String,
+                               completion: @escaping (Result<WCPayPaymentIntent, Error>) -> Void) {
+        let path = "payments/orders/" + String(orderID) + "/capture"
```

Comment:
We tend to declare the paths in a private enum. Also, I would say that the preferred way to compose the path would be by interpolating strings, so in the end, this line could look like:

let path = "\(Paths.ordersPath)/\(orderID)/\(Paths.capture)"

Reply:
Can do

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3998/commits/5680ccc1511a204bfaf30acc8364f44b8ab3c2ff